### PR TITLE
Improve readme with screenshots and details

### PR DESCRIPTION
**Pull request by NeuralAlchemist**
Add proper readme with screenshots and details about setup .

### CODE REVIEW

**Review by eeels22 — APPROVED** [review]
Thanks Yash, looks great

File: README.md [hunk 1]
```diff
@@ -1,37 +1,94 @@
...
+- [Vision](#vision)
+- [Features](#features)
+- [Technologies](#technologies)
+- [Setup](#setup)
+- [Usage](#usage)
+- [Contributing](#contributing)
+
+## Vision
+
+The Cuddly Team believes that pets are not just animals, but that pets are family. We want to help spread the joy that pets bring to our lives and facilitate trusted connections to improve pet care. So we made a social media application that allows pet fans to share experiences and expertise, and grow relationships.
```
Comment (estherchee) [inline README.md:20]: Suggest to mention all types of users, like the following:

So we made a social media application to gather pets parents, pet fans, pet caretakers and pet service providers, empower the users to share experiences and expertise, and grow relationships.

**Review by estherchee — APPROVED** [review]
Great work! Added a minor comment to mention all user types in the vision.

File: README.md (hunk 1 above)
Comment (NeuralAlchemist) [reply README.md:20]: i will add it in the features !

**Pull request merged**